deeppavlov/metrics/fmeasure.py

```diff
@@ -27,3 +30,19 @@ def ner_f1(y_true, y_predicted):
                                   print_results=False)
     f1 = results['__total__']['f1']
     return f1
+
+@register_metric('f1')
+def round_f1(y_true, y_predicted):
+    """
+    Calculates F1 measure.
+
+    Args:
+        y_true: list of true values
+        y_predicted: list of predicted values
+
+    Returns:
+        F1 score
+    """
+    predictions = list(map(lambda x: round(x), y_predicted))
```

Comment:
```python
list(map(round, y_predicted))
```
or
```python
[round(x) for x in y_predicted]
```
round is already a function, no need to create a lambda from it

Reply:
corrected